packages/apps/esm-login-app/src/location-picker/location-picker.component.tsx

```diff
@@ -108,6 +116,11 @@ const LocationPicker: React.FC<LocationPickerProps> = ({ hideWelcomeMessage, cur
         changeLocation(locations[0]?.resource.id, false);
       }
       if (!locations?.length) {
+        showToast({
+          title: t('error', 'Error'),
+          kind: 'error',
+          description: 'No locations were found for this system. Please contact your administrator',
```

Comment:
The description here should also be run through the translation system.
```suggestion
          description: 'No locations were found for this system. Please contact your administrator.',
```

Reply:
@ibacher Just a minor comment: I would suggest adjusting the message to something like: 'No login locations were found for this user or system. Please contact your administrator.' This is because the suggested message seems to imply that there are no locations found in the system, yet it could be either of the following scenarios:

1. No login locations are configured in the system.
3. The current user is not assigned any of the available login locations (relevant in cases where tools like the data-filter module are used. This used to work well with the 2.x login not sure if the filters apply to the way we load locations in O3)

Reply:
@samuelmale That's technically true, but is a user going to understand the difference between "location" and "login location"? The text on the screen always just describes them as "locations".

And, yes, datafilter still works with O3.